lib/model/query/datasets.js

```diff
@@ -268,6 +269,7 @@ const getMetadata = (dataset) => async ({ all, Datasets }) => {
     if (!propertiesMap.has(property.name)) {
       propertiesMap.set(property.name, {
         ...property,
+        name: sanitizeFieldsForOdata ? sanitizeOdataIdentifier(property.name) : property.name,
```

Comment:
Because of the update modal, I think Frontend needs to know both the actual property name and the OData identifier. It needs to know the actual property name so that it can make the correct PATCH request from the update modal in the entities table. Rather than overwriting `name`, what do you think about adding a second property, maybe `odataName`?

Reply:
how about I return `name` and `odataName` in all situations and remove `odata` query parameter?

Reply:
That sounds good to me. :+1: